core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1427,7 +1444,7 @@ public void removeColumn(CharSequence name) {
         final int index = getColumnIndex(name);
         final int type = metadata.getColumnType(index);
 
-        LOG.info().$("removing column '").utf8(name).$("' from ").$(path).$();
+        LOG.info().$("removing column '").utf8(name).$("' from ").utf8(path).$();
```

Comment:
neither is standard way of logging. There is no need to assemble an English sentence here:

```
LOG.info().$("removing [column=`").utf8(name).$(", path=").utf8(path).I$();
```